src/server/test/web/readingsBarGroupFlow.js

```diff
@@ -9,23 +9,112 @@
 
 const { chai, mocha, app } = require('../common');
 const Unit = require('../../models/Unit');
-// const { prepareTest,
-//     parseExpectedCsv,
-//     createTimeString,
-//     expectReadingToEqualExpected,
-//     getUnitId,
-//     ETERNITY,
-//     METER_ID,
-//     unitDatakWh,
-//     conversionDatakWh,
-//     meterDatakWh } = require('../../util/readingsUtils');
+const { prepareTest,
+    parseExpectedCsv,
+    createTimeString,
+    expectReadingToEqualExpected,
+    getUnitId,
+    ETERNITY,
+    METER_ID,
+    GROUP_ID,
+    unitDatakWh,
+    conversionDatakWh,
+    meterDatakWh } = require('../../util/readingsUtils');
 
 mocha.describe('readings API', () => {
     mocha.describe('readings test, test if data returned by API is as expected', () => {
         mocha.describe('for bar charts', () => {
             mocha.describe('for quantity groups', () => {
-
-                // Add BG15 here
+                mocha.it('BG15: expected_bar_group_ri_15-20_mu_kW_gu_kW_st_-inf_et_inf_bd_13.csv', async () =>{
```

Comment:
The string should have the description given in the design doc for this case instead of the file name.